content/knitr/options.md

```diff
@@ -138,7 +138,10 @@ Below is a list of chunk options in **knitr** documented in the format
 -   `warning`: (`TRUE`; logical) Whether to preserve warnings (produced by
     `warning()`) in the output. If `FALSE`, all warnings will be printed in the
     console instead of the output document. It can also take numeric values as
-    indices to select a subset of warnings to include in the output.
+    indices to select a subset of warnings to include in the output. Note that numeric 
+    indexes for warning directly count the number of warnings (e.g. 3 means "the third 
```

Comment:
A question about English (which you certainly know much better than me): which one of `indexes` and `indices` is more widely used for the plural form of `index`? I have been using the latter myself.

Reply:
Thank you -- good point! You are very right that `indices` is more correct and grammatical. I've heard both used in tech concepts, but I think `indices` is definitely clearer (especially since `indexes` is the verb). Once you drew my attention to it, I realize this sentence was somewhat clumsy anyway. I changed the inconsistency and also rewrote it for clarirty.